install/helm/gloo/generate/values.go

```diff
@@ -438,7 +440,8 @@ type GatewayProxyPodTemplate struct {
 	NodeName                      *string               `json:"nodeName,omitempty" desc:"name of node to run on"`
 	NodeSelector                  map[string]string     `json:"nodeSelector,omitempty" desc:"label selector for nodes"`
 	Tolerations                   []*appsv1.Toleration  `json:"tolerations,omitempty"`
-	Probes                        *bool                 `json:"probes,omitempty" desc:"enable liveness and readiness probes"`
+	Probes                        *bool                 `json:"probes,omitempty" desc:"enable readiness probe and allow liveness probe to be enabled"`
+	LivenessProbeEnabled          *bool                 `json:"livenessProbeEnabled,omitempty" desc:"enable liveness probe, Default is false"`
```

Comment:
Is this for a different component than `gloo` or the gateway? If so, we might want to put that.
```suggestion
	LivenessProbeEnabled          *bool                 `json:"livenessProbeEnabled,omitempty" desc:"Set to true to enable a liveness probe (default is false)."`
```